mapdata/WarcraftLegacies/AbilityData/Skin/925910093.json

```diff
@@ -0,0 +1,86 @@
+{
+  "OldId": 1986086977,
+  "NewId": 925910093,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Id": 1718840929,
+      "Type": 3,
+      "Value": "(Yellow-Muradin)"
+    },
+    {
+      "Id": 2037082209,
+      "Type": 3,
+      "Value": "R"
+    },
+    {
+      "Id": 1802007137,
+      "Type": 3,
+      "Value": "R"
+    },
+    {
+      "Id": 1952805473,
+      "Type": 3,
+      "Value": "Learn Perfect Avatar - [|cffffcc00Level %d|r]"
```

Comment:
I think actually let's just keep this ability named Avatar. The reason the unit version had Perfect in the name was because it had a ludicrously high armor value, so if we're not keeping that (which is fine) then this can just be Avatar. It's the same ability!

Reply:
Muradin has normal vanilla Avatar (+dmg , +HP. +Armor, +Spell Immune as his ult) with same name and icon as this now. Maybe just swap them and scale the durations / effects, rather than Thanes having a watered down version. This way we keep both abilities in the map!